source/cli.ts

```diff
@@ -8,20 +8,49 @@ const cli = meow(`
 	Usage
 		$ tsd [path]
 
+	Options
+		--verify-size, -s  Keep on top of the memory usage with a data snapshot
+		--write, -w        Overwrite existing memory usage JSON fixture
+		--size-delta, -d   What percent of change is allow in memory usage, default is 10
```

Comment:
Kinda seems like these should be in a subcommand, like `tsd memory-usage --write`. For example, `tsd --write` is way too generic naming for the top-level interface. However, we can't really do subcommands as we expect paths. Alternatively, we could namespace the flags: `--mem-write`, `--mem-verify-size`, ...

Reply:
@sindresorhus We could however do `tsd --cwd=path` instead of `tsd path` so we can still accept subcommands?

Or `tsd verify path` or something? Not sure, just thinking out loud.

Reply:
I'd like to ideally make it run at the same time as normal tests (rather than have to run it as a 2nd command)  - which is why I opted for using flags on the main command. 

I copied `--write` from jest, but I'm open to having it more verbose and specific too 👍